perllib/FixMyStreet/Reporting.pm

```diff
@@ -18,7 +18,7 @@ has on_updates => ( is => 'lazy', default => sub { $_[0]->type eq 'updates' } );
 
 has body => ( is => 'ro', isa => Maybe[InstanceOf['FixMyStreet::DB::Result::Body']] );
 has wards => ( is => 'ro', isa => ArrayRef[Int], default => sub { [] } );
-has category => ( is => 'ro', isa => Maybe[Str] );
+has category => ( is => 'ro', isa => Maybe[ArrayRef[Maybe[Str]]], default => sub { [] } );
```

Comment:
Feels like we could get rid of these Maybes, the code changes in this file could be easier, e.g. if we coerced the category to always be an ArrayRef[Str], then you wouldn't need to check $self->category is set anywhere, it always would be an arrayref.